indico/modules/events/registration/models/registrations.py

```diff
@@ -466,6 +467,10 @@ def is_ticket_blocked(self):
         """Check whether the ticket is blocked by a plugin."""
         return any(values_from_signal(signals.event.is_ticket_blocked.send(self), single_value=True))
 
+    @property
+    def google_wallet_ticket_id(self):
```

Comment:
Is this, technically, `google_wallet_ticket_object_id`?

Reply:
Yes, it is. Not sure if I want this verbosity here though... :P